integration_tests/sdk/setup_integration.py

```diff
@@ -325,6 +325,24 @@ def _fetch_integration_credentials(section: str, name: str) -> Dict[str, Any]:
     return test_credentials[section][name]
 
 
+def lazy_configuration(name: str) -> bool:
+    """
+    Checks config to see if given compute integration should have lazy exeuction.
+    Returns True if if given compute integration has `enable_previews` set.
+    Returns False otherwise.
+    """
+    test_credentials = _parse_credentials_file()
+
+    assert "compute" in test_credentials, "compute section expected in test-credentials.yml"
+    assert name in test_credentials["compute"].keys(), "%s not in test-credentials.yml." % name
+
+    if "enable_previews" in test_credentials["compute"][name].keys():
```

Comment:
Maybe we can just `return "enable_previews" not in test_credentials["compute"][name].keys()`. It should be pretty clear what is being returned from the docstring.